app/components/transactions/TransactionsMeta.tsx

```diff
@@ -1,7 +1,7 @@
-import React from 'react';
-import styled from 'styled-components';
-import { formatSmidge } from '../../infra/utils';
-import { smColors } from '../../vars';
+import React from "react";
+import styled from "styled-components";
+import { formatSmidge } from "../../infra/utils";
+import { smColors } from "../../vars";
```

Comment:
Can u adjust the eslint/prettier based on the project, u can do the `yarn run lint `.

Reply:
Done